postgresql-ballerina/Module.md

```diff
@@ -0,0 +1,386 @@
+## Overview
+
+This module provides the functionality required to access and manipulate data stored in a PostgreSQL database.
+
+**Prerequisite:** Add the PostgreSQL driver JAR as a native library dependency in your Ballerina project.
```

Comment:
You can either download the JAR manually and config the path or config the maven dependency config in `Ballerina.toml` file as mentioned below.